src/owasp-dependency-check-suppressions.xml

```diff
@@ -37,7 +37,19 @@
         <vulnerabilityName regex="true">.*</vulnerabilityName>
     </suppress>
 
-    <!-- see https://github.com/apache/pulsar/pull/14629-->
+    <!-- see https://github.com/apache/pulsar/pull/16110 -->
+    <suppress>
+        <notes>CVE-2022-23712 is only related to Elastic server</notes>
+        <gav>co.elastic.clients:elasticsearch-java:8.1.0</gav>
```

Comment:
could you use the sha1 here?

Reply:
Hi @nicoloboschi what's the different between `gav` and `sha1`, or what's the benefit to use `sha1`? Also, how can I get the corresponding `sha1` of this artifact?

Reply:
It seems that I can get the value of `sha1` from https://repo1.maven.org/maven2/co/elastic/clients/elasticsearch-java/8.1.0/elasticsearch-java-8.1.0.jar.sha1 and ditto rest-client.

I'll make an update but still wonder what's the benefit it brings.

Reply:
Actually I open the OWASP html report and click on the "add suppression" button to get the SHA-1.

Reply:
Thanks for your information! I've addressed this comment.